### PR TITLE
[Crypto] Expose `OS.get_entropy`

### PR DESCRIPTION
**Pull request by Faless**

### CODE REVIEW

File: doc/classes/OS.xml [hunk 1]
```diff
@@ -262,6 +262,14 @@
 				[b]Note:[/b] This method is not supported on the Web platform. It returns an empty string.
 			</description>
 		</method>
+		<method name="get_entropy">
+			<return type="PackedByteArray" />
+			<param index="0" name="size" type="int" />
+			<description>
+				Retrieves a [PackedByteArray] of entropy with given [param size] from the operating system.
```
Comment (Calinou) [inline doc/classes/OS.xml:269]: Does "of given size" refer to the array size or the entropy? If it refers to the array size, what does "of entropy" refer to? Usually, I hear about the terms high/low entropy (or defined in a number of bits).

Comment (AThousandShips) [reply doc/classes/OS.xml:269]: In this case it refers to the entropy data used by the OS to generate random numbers, but unsure how to describe it here, maybe just "of entropy data" would be sufficient

Comment (Faless) [reply doc/classes/OS.xml:269]: I agree this is quite confusing, I think we can just reuse what we have in generate_random_bytes:
```
Generates a PackedByteArray of cryptographically secure random bytes with given size.
```

File: doc/classes/OS.xml [hunk 2]
```diff
@@ -262,6 +262,14 @@
 				[b]Note:[/b] This method is not supported on the Web platform. It returns an empty string.
 			</description>
 		</method>
+		<method name="get_entropy">
+			<return type="PackedByteArray" />
+			<param index="0" name="size" type="int" />
+			<description>
+				Generates a [PackedByteArray] of cryptographically secure random bytes with given [param size].
+				[b]Note:[/b] Generating large quantity of bytes using this method can result in locking and entropy of lower quality on most platforms. Using [method Crypto.generate_random_bytes] is preferred in most cases.
```
Comment (AThousandShips) [inline doc/classes/OS.xml:270]:
```suggestion
				[b]Note:[/b] Generating a large quantity of bytes using this method can result in locking and entropy of lower quality on most platforms. Using [method Crypto.generate_random_bytes] is preferred in most cases.
```
or
```suggestion
				[b]Note:[/b] Generating large quantities of bytes using this method can result in locking and entropy of lower quality on most platforms. Using [method Crypto.generate_random_bytes] is preferred in most cases.
```

**Review by AThousandShips — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Needs rebase.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**